packages/charts/theme/material/vaadin-chart-styles.js

```diff
@@ -0,0 +1,81 @@
+      --vaadin-charts-background: var(--material-background-color);
+      --vaadin-charts-title-label: var(--material-body-text-color);
+      --vaadin-charts-axis-title: var(--material-secondary-text-color);
+      --vaadin-charts-axis-label: var(--material-secondary-text-color);
+      --vaadin-charts-data-label: var(--material-body-text-color);
+      --vaadin-charts-secondary-label: var(--material-secondary-text-color);
+      --vaadin-charts-axis-line: var(--vaadin-charts-contrast-5pct);
+      --vaadin-charts-xaxis-line-width: 0;
+      --vaadin-charts-grid-line: var(--vaadin-charts-contrast-10pct);
+      --vaadin-charts-disabled-label: var(--material-disabled-text-color);
+      --vaadin-charts-contrast: var(--material-body-text-color);
+      --vaadin-charts-contrast-5pct: var(--material-secondary-background-color);
+      --vaadin-charts-contrast-10pct: var(--material-divider-color);
+      --vaadin-charts-contrast-20pct: var(--material-disabled-color);
+      --vaadin-charts-contrast-60pct: var(--material-secondary-text-color);
+      --vaadin-charts-tooltip-background: var(--material-background-color);
+      --vaadin-charts-tooltip-background-opacity: 1;
+      --vaadin-charts-tooltip-border-color: var(--material-background-color);
+      --vaadin-charts-button-label: var(--material-primary-color);
+      --vaadin-charts-button-background: var(--material-background-color);
+      --vaadin-charts-button-hover-background: var(--material-secondary-background-color);
+      --vaadin-charts-button-active-label: var(--material-primary-contrast-color);
+      --vaadin-charts-button-active-background: var(--material-primary-color);
+    }
+
+    :host([theme~='gradient']) {
+      --vaadin-charts-color-0: #6200ee;
+      --vaadin-charts-color-1: #6962e8;
+      --vaadin-charts-color-2: #619fe1;
+      --vaadin-charts-color-3: #2fbfd3;
+      --vaadin-charts-color-4: #03dac6;
+      --vaadin-charts-color-5: #5bd091;
+      --vaadin-charts-color-6: #acc65e;
+      --vaadin-charts-color-7: #ffbc2b;
+      --vaadin-charts-color-8: #fe9940;
+      --vaadin-charts-color-9: #fd695e;
+      --vaadin-charts-color-positive: var(--vaadin-charts-color-6);
+      --vaadin-charts-color-negative: var(--vaadin-charts-color-9);
+    }
+
+    :host([theme~='monotone']) {
+      --vaadin-charts-color-0: #6200ee;
+      --vaadin-charts-color-1: #7f39fb;
+      --vaadin-charts-color-2: #985eff;
+      --vaadin-charts-color-3: #bb86fc;
+      --vaadin-charts-color-4: #dbb2ff;
+      --vaadin-charts-color-5: #ecddfd;
+      --vaadin-charts-color-6: #9fe3eb;
+      --vaadin-charts-color-7: #03dac6;
+      --vaadin-charts-color-8: #00c4b4;
+      --vaadin-charts-color-9: #01a299;
+      --vaadin-charts-color-positive: var(--vaadin-charts-color-8);
+      --vaadin-charts-color-negative: var(--vaadin-charts-color-0);
+    }
+
+    .highcharts-container {
+      font-family: var(--material-font-family, 'Roboto', sans-serif);
```

Comment:
`--material-font-family` should always be defined. That said, this file is missing imports for the `color` and `typography` style sheets from material-styles (it now assumes those custom properties are available).


```suggestion
      font-family: var(--material-font-family);
```

Reply:
I applied this suggestion along with an import to `@vaadin/vaadin-material-styles/typography.js`